client/subscriptions.go

```diff
@@ -0,0 +1,64 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"github.com/lithictech/webhookdb-cli/types"
+)
+
+type SubscriptionInfoInput struct {
+	AuthCookie    types.AuthCookie    `json:"-"`
+	OrgIdentifier types.OrgIdentifier `json:"identifier"`
+}
+
+type SubscriptionInfoOutput struct {
+	OrgName          string `json:"org_name"`
+	BillingEmail     string `json:"billing_email"`
+	IntegrationsUsed string `json:"integrations_used"`
+	PlanName         string `json:"plan_name"`
+	IntegrationsLeft string `json:"integrations_left"`
+	SubStatus        string `json:"sub_status"`
+}
+
+func SubscriptionInfo(c context.Context, input SubscriptionInfoInput) (out SubscriptionInfoOutput, err error) {
```

Comment:
Let's remove the named outputs and use `var out SubscriptionInfoOutput`, to avoid the ambiguous `err` variable reassignment.